### DISCUSSION TITLE
Is it possible to obtain current playback time via command line or AppleScript (on macOS)?

### DISCUSSION THREAD

**Discussion by patricorgi** [iina/iina#3697] [Q&A] [answered]
When playing a video, if I'd like to take bullet notes at a moment, is it possible to get current playback time through command line? If possible, I can automate this note-taking process. Thank you in advance!

**Comment by low-batt** [comment, marked as answer]
Yes. See my answer to issue #3370

**Reply by patricorgi** [reply]
Thank you!

**Reply by ELgit** [reply]
I posted a question in the issue #3370 link. I don't know the steps (not a programmer) to invoke the described commands and bind a key command shortcut to insert text string of playback time into the app being used to transcribe. Thanks in advance for any help!
